src/platforms/java/migration.mdx

```diff
@@ -4,6 +4,11 @@ sidebar_order: 1000
 description: "Learn about migrating from io.sentry:sentry 1.x to io.sentry:sentry 3.x."
 ---
 
+## Migrating from `io.sentry:sentry` `4.3.0` to `io.sentry:sentry` `4.4.0`
+
+`Sentry#startTransaction` by default binds created transaction to the scope. Bound transaction can be retrieved with `Sentry#getSpan`.
+To start transaction without binding to the scope, use one of the new overloaded `startTransaction` methods taking `bindToScope` parameter and set it to `false`.
```

Comment:
the opposite, right?
maybe this should be a separated PR, ideally, we'd get this merged before going 5.0 GA so people know the breaking changes in advance

Reply:
👍